macro/duplicateHits.C

```diff
@@ -23,6 +23,7 @@
 
 #ifdef ENABLE_UPGRADES
 #include "EndCapsSimulation/Hit.h"
+#include "ForwardTrackerSimulation/Hit.h"
```

Comment:
I have no clue if I am being honest. I copied the O2 around two months ago and then this might have been there and a later pull request might have removed it (and so this older version is trying to place it back). I will look for the pull request which removed it.

Reply:
Removed